docs/runtimes/configuring-job-resources.md

```diff
@@ -113,12 +113,12 @@ Node selector is supported for all cloud platforms. It is relevant for MLRun and
 
 When running ML functions you might want to control whether to run on spot nodes or on-demand nodes. Preemption mode controls whether pods can be scheduled on preemptible (spot) nodes. Preemption mode is supported for all functions. 
 
-Preemption mode uses Kubernets Taints and Toleration to enforce the mode selected. Read more in [Kubernetes Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration). 
+Preemption mode uses Kubernetes Taints and Toleration to enforce the mode selected. Read more in [Kubernetes Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration). 
 
 ### Why preemption mode
 
 On-demand instances provide full control over the instance lifecycle. You decide when to launch, stop, hibernate, start, 
-reboot, or terminate it. With Spot instances you request capacity from specific available zones, though it is  
+reboot, or terminate it. With Spot instances, you request capacity from specific availability zones, though it is
```

Comment:
*from specific available zones*

Reply:
Hi @jillnogold, I am not sure if your comment is a correction or just stating what it said before. Would you like me to revert [availability zones](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) back to `available` zones?

Reply:
Hi @bcrant - Thanks for your input! And please ignore that comment.